src/Cassandra/UdtMappingDefinitions.cs

```diff
@@ -89,7 +90,8 @@ private async Task<UdtColumnInfo> GetDefinitionAsync(string keyspace, UdtMap map
             if (map.IgnoreCase)
             {
                 //identifiers are lower cased in Cassandra
-                caseSensitiveUdtName = caseSensitiveUdtName.ToLower();
+                CultureInfo textInfo = new CultureInfo("en-US");
+                caseSensitiveUdtName = textInfo.TextInfo.ToLower(caseSensitiveUdtName);
```

Comment:
Can we use `ToLowerInvariant()`, that uses [`InvariantCulture`](https://docs.microsoft.com/en-us/dotnet/api/system.globalization.cultureinfo.invariantculture) instead?

Reply:
I tried `ToLowerInvariant()` but it didn't work.

Reply:
[`ToLowerInvariant()` calls `ToLower(CultureInfo.InvariantCulture)` internally](https://referencesource.microsoft.com/#mscorlib/system/string.cs,2662). It should work, could you try again?

We should use either `ToLowerInvariant()` or `ToLower(CultureInfo.InvariantCulture)` and avoid using a new `CultureInfo("en-US")` instance.

Reply:
Sorry, it was my mistake. I tried both of them again and `ToLowerInvariant()` works. `ToLower(CultureInfo.InvariantCulture)` also works.